### ISSUE TITLE
Support for free plan

### ISSUE THREAD

**Issue by ampilogov** [homebridge-simplisafe3/homebridge-simplisafe3#315]
### Is your feature request related to a problem? Please describe:
Not working with free plan

### Describe the solution you'd like:
I have simplisafe with a free plan (Basic App Control). I'm able to arm/disarm system and monitor sensors from my iOS app or web. Can we use the same API to connect it to homebrige?

### Describe alternatives you've considered:
Buy 10$ subscription

**Comment by shamoon** [comment]
Its not up to us, the paid plans provide access to the API. Second line in the readme =) https://github.com/homebridge-simplisafe3/homebridge-simplisafe3#requirements

**Comment by ampilogov** [comment]
Why we can just send the same request as on web. For example to disable alarm we need to send:

https://api.simplisafe.com/v1/ss3/subscriptions/5518648/state/off

**Comment by shamoon** [comment]
Have you tested it works with the free plan? I do not believe thats the case but I suppose things could have changed. But then why would anyone pay for the $10 plan? The app uses the same API

**Comment by ampilogov** [comment]
But then why would anyone pay for the $10 plan? - to store video recordings on motion

**Comment by ampilogov** [comment]
Have you tested it works with the free plan? - testing it right now on my free plan

**Comment by shamoon** [comment]
Oh wow, I guess thats a change. I remember hearing about this.

Change line 298 of `simplisafe.js` to `let subscriptions = data.subscriptions.filter(s => s.activated > 0);` and restart. Let me know the result. 

But Im not sure what will / wont work e.g. it says you dont get notifications which I assume would mean none of the events would come through (websocket). If thats true then when the system is armed from elsewhere the plugin wont know this, cameras wont work properly, nor will motion sensors. So it'd be a pretty hobbled plugin, which is only going to cause confusion and frustration.

If thats not true then Im all for it.

**Comment by ampilogov** [comment]
The subscription request returns empty result, I don't think the change on line 298 will help.

**Comment by ampilogov** [comment]
@shamoon Removing "&& s.activated > 0)" from this line fix the problem https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/blob/05d11138b3611192d2b559a0556fb90412eaf144/src/simplisafe.js#L236

This change was introduced in 1.9.5. Could we revert it in the next release?

**Comment by shamoon** [comment]
Can you send me the output of `this.log(subscriptions)` (just ** out anything private). I can't test this myself.

So you're saying *everything* works? Including motion sensor / camera motion events, alarm notifications (eg alarm triggers and HK gets updated)?

**Comment by ampilogov** [comment]
```
{
    uid: 4XXXXX5,
    sid: 5XXXXX8,
    sStatus: 7,
    activated: 0,
    planSku: 'SSBAPP',
    planName: 'Basic App Control',
    price: 0,
    currency: 'USD',
    country: 'US',
    expires: 0,
    canceled: 0,
    extraTime: 0,
    creditCard: { lastFour: '', type: '', ppid: null, backupPpid: null, uid: -1 },
    time: 2628000,
    paymentProfileId: null,
    features: {
      monitoring: false,
      alerts: true,
      online: true,
      hazard: false,
      video: true,
      cameras: 0,
      dispatch: false,
      proInstall: false,
      discount: 0,
      vipCS: false,
      medical: false,
      careVisit: false,
      storageDays: 0
    },
    status: { hasBaseStation: true, isActive: true, monitoring: 'None' },
    subscriptionFeatures: {
      monitoredSensorsTypes: [],
      monitoredPanicConditions: [],
      dispatchTypes: [],
      remoteControl: [Array],
      cameraFeatures: [Object],
      support: [Object],
      cellCommunicationBackup: false,
      alertChannels: [Array],
      alertTypes: [Array],
      alarmModes: [Array],
      supportedIntegrations: [],
      timeline: [Object],
      isProfessionallyMonitored: false
    },
    dispatcher: 'cops',
    dcid: 0,
    activationCode: '',
    systemVersion: 20,
    orderProductId: 0,
    location: {
      sid: 5XXXX8,
      uid: 4XXXXX5,
      lStatus: 0,
      account: '0XXXX9',
      street1: '',
      street2: '',
      locationName: '2XXXXXXXe',
      city: '',
      county: '',
      zip: '',
      country: 'US',
      crossStreet: '',
      notes: '',
      residenceType: 0,
      numAdults: -1,
      numChildren: -1,
      locationOffset: -300,
      safeWord: '',
      signature: '',
      timeZone: 0,
      primaryContacts: [Array],
      secondaryContacts: [Array],
      copsOptIn: false,
      certificateUri: 'hXXXXXX8',
      nestStructureId: '',
      system: [Object],
      state: ''
    },
    pinUnlocked: true,
    pinUnlockedBy: 'bluetooth',
    billDate: 0,
    billInterval: 2628000,
    autoActivation: null
  },
  {
    uid: 4XXXXX5,
    sid: 5XXXXX7,
    sStatus: 0,
    activated: 0,
    planSku: 'NEWUSER',
    planName: 'New User Plan',
    price: 0,
    currency: 'USD',
    country: 'US',
    expires: 0,
    canceled: 0,
    extraTime: 5256000,
    creditCard: {
      lastFour: '',
      type: '',
      ppid: null,
      backupPpid: null,
      uid: 4996555
    },
    time: 2628000,
    paymentProfileId: null,
    features: {
      monitoring: false,
      alerts: false,
      online: false,
      hazard: false,
      video: false,
      cameras: 0,
      dispatch: false,
      proInstall: false,
      discount: 0,
      vipCS: false,
      medical: false,
      careVisit: false,
      storageDays: 0
    },
    status: {
      hasBaseStation: false,
      isActive: false,
      monitoring: 'None',
      inactiveReason: 'New'
    },
    subscriptionFeatures: {
      monitoredSensorsTypes: [],
      monitoredPanicConditions: [],
      dispatchTypes: [],
      remoteControl: [Array],
      cameraFeatures: [Object],
      support: [Object],
      cellCommunicationBackup: false,
      alertChannels: [],
      alertTypes: [],
      alarmModes: [Array],
      supportedIntegrations: [],
      timeline: [Object],
      isProfessionallyMonitored: false
    },
    dispatcher: 'cops',
    dcid: 0,
    activationCode: 'yXXxXX9',
    systemVersion: 20,
    orderProductId: 0,
    location: {
      sid: 5487887,
      account: '',
      lStatus: 0,
      street1: '',
      street2: '',
      locationName: '',
      city: '',
      state: '',
      zip: '',
      county: '',
      notes: '',
      residenceType: 0,
      numAdults: 0,
      numChildren: 0,
      safeWord: '',
      signature: '',
      primaryContacts: [Array],
      secondaryContacts: [Array],
      copsOptIn: false,
      certificateUri: '',
      locationOffset: 0,
      system: [Object]
    },
    pinUnlocked: false,
    pinUnlockedBy: '',
    billDate: 5256000,
    billInterval: 2628000,
    autoActivation: null
  }
]
```


Yep I'm getting notifications to my iPhone. Can't test camera since it's broken, but everything else works. https://ibb.co/nLZL3Xq

**Comment by shamoon** [comment]
Thank you. Oh so now I understand, this was working for you up until the most recent version? The free plan is never `activated`. I will need to update the documentation too.

**Comment by ampilogov** [comment]
Yep, I've installed previous version (1.9.4) and it's works. After removing the line (above) it also works in 1.9.5.

**Comment by shamoon** [comment]
Should be fixed in 1.9.6

**Comment by shamoon** [comment]
Grr, 1.9.7
